com.brave.Brave.metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<!-- Copyright 2017 the Chromium authors, with modifications -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.brave.Brave</id>
```

Comment:
Just as another data point, Android seems to use `com.brave.browser`. Would be good to settle on something with upstream to avoid renames in the future.

Reply:
cc @fmarier

Reply:
`com.brave.browser` sounds good to me.

Reply:
Should I capitalize `browser` so it becomes `com.brave.Browser`?

Reply:
@fmarier Could Brave approve and maintain this?